selftests/deployment/deployment.yml

```diff
@@ -5,6 +5,7 @@
     - vars.yml
   roles:
     - common
+    - cirrus
```

Comment:
I think automating this is very valuable, but, I don't see how it fits into the "deployment" checks besides the fact that the manual release test plan is about deployment to a large extent.  This would, for instance, run on all different distros we want to test the deployment, but it's nothing that depends on those distros.  In fact, I was expecting this to use a `delegate_to: localhost`.

What if we add another playbook just for checking statuses?  I think it'd be fairer and would avoid polluting the deployment ones with unrelated checks.

Reply:
BTW, one other possibility, beyond a playbook, is to write tests similar to https://github.com/avocado-framework/avocado/blob/master/selftests/functional/test_thirdparty_bugs.py#L39 , maybe keeping them at a different directory (`selftests/service_statuses/` ?).

Reply:
@clebergnu I see your point and makes sense.

My only concern is to not remove one additional step from our "pre-release test plan". If we create another playbook or a different script (not sure if we need both), we need to think on how we are going to put this into our pipeline to avoid having an additional step on our "pre-release test plan", like, for instance: "Run our service status check".

What do you think? Is that possible to have this into our pipeline, like on our selftests scripts? So, `make check` will take care of this?